src/main/java/com/mixpanel/android/viewcrawler/ViewVisitor.java

```diff
@@ -282,6 +251,51 @@ private boolean verifyLayout(View target, Set<Integer> rules) {
                 }
             }
 
+            int size = mArgs.size();
+            for (int i = 0; i < size; i++) {
+                LayoutRule layoutRule = mArgs.get(i);
+                final int viewId = layoutRule.viewId;
```

Comment:
In Javascript, we always try to assign a value in a dictionary to a variable if such thing is used more than once. In Java, do we have a different pattern?